catalog.html

```diff
@@ -113,10 +113,10 @@ <h2 class="visually-hidden">Выбор гостиниц по фильтрам</h
       </div>
         <section class="about-hotels">
           <h2 class="catalog-head">Найдено гостиниц: 38</h2>
-          <select class="sorting-by-price"aria-label="Сортировка">
-            <option class="sorting-by-price" value="cheap" selected>Сначала дешёвые</option>
-            <option class="sorting-by-price" value="expensive">Сначала дорогие</option>
-            <option class="sorting-by-price" value="popular">Сначала популярные</option>
+          <select class="sorting"aria-label="Сортировка">
+            <option class="sorting" value="cheap" selected>Сначала дешёвые</option>
+            <option class="sorting" value="expensive">Сначала дорогие</option>
+            <option class="sorting" value="popular">Сначала популярные</option>
           </select>
```

Comment:
```suggestion
          <select class="sorting" aria-label="Сортировка">
            <option value="cheap" selected>Сначала дешёвые</option>
            <option value="expensive">Сначала дорогие</option>
            <option value="popular">Сначала популярные</option>
          </select>
```
Забыл пробел. 
Я бы не стала задавать одинаковые классы селекту и его элементам. А сейчас нам вообще не нужны классы для option, мы не собираемся их стилизовать.